Yank/experiment.py

```diff
@@ -2662,8 +2734,7 @@ def _build_experiment(self, experiment_path, experiment):
 
         # Get and validate experiment sub-options and divide them by class.
         exp_opts = self._determine_experiment_options(experiment)
-        (exp_opts, phase_opts, sampler_opts,
-         alchemical_region_opts, alchemical_factory_opts) = exp_opts
+        (exp_opts, phase_opts, alchemical_region_opts, alchemical_factory_opts) = exp_opts
```

Comment:
Since the sampler have been stripped out of here, do the `!Combinatorial` experiments still work, I'm assuming so, just checking.

Reply:
I think they should work as usual! I haven't added the possibility to have combinatorial samplers yet, but I thought we can add that later in a second pass.